cypress-tests/cypress/support/pageBuilder/pbUpdatePage.ts

```diff
@@ -0,0 +1,61 @@
+import { gqlClient } from "../utils";
+
+interface PbUpdatePageInput {
+    id: string;
+    data: {
+        category?: string;
+        path?: string;
+        title?: string;
+        settings?: {
+            general?: {
+                layout?: string;
+                tags?: string[];
+            };
+        };
+    };
+}
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbUpdatePage(data: PbUpdatePageInput): Promise<string>;
```

Comment:
In L58, we're returning `data` which is an object, not string.

Let's switch `string` in `Promise<string>` to an object that consists of keys defined in the GQL query, starting L32.